### PR TITLE
refactor(preferenceslist infocard): v2 style

### PR DESCRIPTION
**Pull request by kramduckner**
# Pull Request Template

## Issue Overview

Bringing the PreferenceList and InfoCard components into the world of [v2 styling](https://github.com/bloom-housing/bloom/blob/main/docs/Styling2ndGen.md).

This PR addresses #issue

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Refactored both PreferenceList and InfoCard components using the v2 style guidelines.

## How Can This Be Tested/Reviewed?

Compare storybooks, app instances, and code for any visual or functional regressions. Ensure that the code changes follow the [v2 styling](https://github.com/bloom-housing/bloom/blob/main/docs/Styling2ndGen.md). Check that the new docs are correct.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [ ] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Comment by emilyjablonski** [comment]
[Storybook Preview](https://63877eb9a505d40008ab9332--amazing-davinci-96c1cd.netlify.app/?path=/story/documentation-utilities--page)

File: ui-components/src/blocks/InfoCard.docs.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { Canvas, Story, ArgsTable } from "@storybook/addon-docs"
+import { Swatch } from "../prototypes/Swatch"
+import { InfoCard } from "./InfoCard"
+
+# Info Card
+
+The Info Card component renders a box with a title that can be text of a link. Beneath the title is an optional subtitle. Beneath the subtitle can be a string, a component, or markdown.
```
Comment (emilyjablonski) [inline ui-components/src/blocks/InfoCard.docs.mdx:7]: `can be text of a link` might be a typo here?

**Review by emilyjablonski — APPROVED** [review]
This looks great!! My only comment is that there seems to be some spacing issues in the scss files, maybe prettier wasn't running (or maybe that doesn't make its way into emacs as easily?) - I would have expected spaces after the colons.
 🎸

**Comment by emilyjablonski** [comment]
@jaredcwhite Would love your eyes here

File: ui-components/src/blocks/InfoCard.docs.mdx (hunk 1 above)
Comment (kramduckner) [reply ui-components/src/blocks/InfoCard.docs.mdx]: fixed!

**Comment by kramduckner** [comment]
> This looks great!! My only comment is that there seems to be some spacing issues in the scss files, maybe prettier wasn't running (or maybe that doesn't make its way into emacs as easily?) - I would have expected spaces after the colons. 🎸

I think I fixed the spacing! Emacs supports it : ). Just didn't configure it properly.

File: ui-components/src/blocks/InfoCard.scss [hunk 2]
```diff
@@ -1,51 +1,47 @@
...
-  @apply uppercase;
-  @apply font-sans;
-  @apply font-bold;
-  @apply text-sm;
-  @apply tracking-wide;
+  color: var(--title-color);
+  text-transform: var(--title-text-transform);
+  font-weight: bold;
+  font-size: var(--title-font-size);
+  font-family: var(--bloom-font-sans);
```
Comment (ludtkemorgan) [inline ui-components/src/blocks/InfoCard.scss:39]: should this be `var(--title-font-family)`?

File: ui-components/src/lists/PreferencesList.scss [hunk 3]
```diff
@@ -1,6 +1,24 @@
+.preferences-list {
+  --background-color: var(--bloom-color-white);
```
Comment (ludtkemorgan) [inline ui-components/src/lists/PreferencesList.scss:2]: I'm not seeing this one or the `info-card-title-text-transform` used anywhere

**Comment by jaredcwhite** [comment]
This looks great in spirit…my only concern is at a high level, which is that we're auditing all the card-like components across the system and one of the recommendations would be to remove all existing cards and replace with, essentially, Card—whether that ends up being some variants of Card still in the design system like an Info Card, or just an app-specific variants outside of the design system. Thoughts?

**Comment by emilyjablonski** [comment]
@jaredcwhite This is helpful for SF in the short term, can we wait until the audit is finished to consider changes like that?

File: ui-components/src/lists/PreferencesList.scss (hunk 3 above)
Comment (kramduckner) [reply ui-components/src/lists/PreferencesList.scss:2]: removed the --info-card-title-text-transform! good catch.

the --bloom-color-white comes from [colors.scss](https://github.com/bloom-housing/bloom/blob/main/ui-components/src/global/tokens/colors.scss)

File: ui-components/src/blocks/InfoCard.scss (hunk 2 above)
Comment (kramduckner) [reply ui-components/src/blocks/InfoCard.scss]: yup! fixed

Comment (kramduckner) [reply ui-components/src/blocks/InfoCard.scss]: yup! fixed. Thanks.

File: ui-components/src/lists/PreferencesList.scss (hunk 3 above)
Comment (ludtkemorgan) [reply ui-components/src/lists/PreferencesList.scss:2]: The --background-color variable is not being used in this file.

**Review by ludtkemorgan — COMMENTED** [review]
Thanks for making those changes!

File: ui-components/src/lists/PreferencesList.scss (hunk 3 above)
Comment (kramduckner) [reply ui-components/src/lists/PreferencesList.scss]: 🤦 ofc. fixed!

**Review by ludtkemorgan — APPROVED** [review]
Sorry, I thought I approved this

**Pull request merged**